tools/db_bench_tool.cc

```diff
@@ -1055,6 +1076,8 @@ static enum RepFactory StringToRepFactory(const char* ctype) {
 static enum RepFactory FLAGS_rep_factory;
 DEFINE_string(memtablerep, "skip_list", "");
 DEFINE_int64(hash_bucket_count, 1024 * 1024, "hash bucket count");
+DEFINE_double(data_block_hash_table_util_ratio, 0.75,
+              "util ratio for data block hash index table");
```

Comment:
You can also mention that this takes affect only when data_block_index_type is set to "binary_and_hash".